models/global-types.ts

```diff
@@ -251,3 +251,13 @@ export interface CreateEmployeeModalProps {
   formErrors: CreateEmployeeItems;
   setFormErrors: (item: any) => void;
 }
+
+export interface TSignupPayload {
```

Comment:
Remove the `T` from the beginning of this interface, it should looks like ` SignupPayload`